release/sil/sil.tea.dict/source/sil.tea.dict.model.ts

```diff
@@ -0,0 +1,14 @@
+/*
+  dict 1.0 generated from template.
+  
+  This is a minimal lexical model source that uses a tab delimited wordlist.
+  See documentation online at https://help.keyman.com/developer/ for
+  additional parameters.
+*/
+
+const source: LexicalModelSource = {
+  format: 'trie-1.0',
+  wordBreaker: 'default',
+  sources: ['wordlist.tsv'],
+};
```

Comment:
I've tried setting `languageUsesCasing: true` but I'm not seeing a difference...